test/e2e/crds/consumer.go

```diff
@@ -67,7 +90,7 @@ spec:
     filters:
     - type: ExtensionRef
       extensionRef:
-        group: gateway.api7.io
+        group: gateway.apisix.io
```

Comment:
Are all our resources grouped as `gateway.apisix.io`, or are some grouped as `gateway.api7.io` and some grouped as `gateway.apisix.io`?